docs/nidcpower/class.rst

```diff
@@ -3,38 +3,26 @@
 Session
 =======
 
-.. py:class:: Session(self, resource_name, channels=None, reset=False, options={})
+.. py:class:: Session(self, resource_name, channels=None, reset=False, options={}, independent_channels=True)
 
     
 
-    Creates and returns a new NI-DCPower session to the power supply or SMU
-    specified in **resource name** to be used in all subsequent NI-DCPower
-    method calls. With this method, you can optionally set the initial
-    state of the following session properties:
+    Creates a new NI-DCPower session to the specified instrument(s) and channel(s) and returns a
+    session handle to be used in all subsequent NI-DCPower method calls.
```

Comment:
The old docs were Python-specific, this is worse (copied over from C, looks like).